ext/json/json.c

```diff
@@ -178,6 +178,24 @@ PHP_JSON_API zend_result php_json_decode_ex(zval *return_value, const char *str,
 }
 /* }}} */
 
+/* {{{ */
+PHP_JSON_API zend_result php_json_validate_ex(const char *str, size_t str_len, zend_long options, zend_long depth)
+{
+	php_json_parser parser;
+	zval tmp;
+	const php_json_parser_methods parser_validate_methods = get_validate_methods();
```

Comment:
```suggestion
	const php_json_parser_methods parser_validate_methods = php_json_get_validate_methods();
```

Reply:
it might be actually better to pass the pointer rather than do structure copy.

Reply:
it could also return const pointer instead

Reply:
:heavy_check_mark: